tests/sentinel/tests/14-debug-command.tcl

```diff
@@ -0,0 +1,9 @@
+source "../tests/includes/init-tests.tcl"
+
+test "Sentinel debug test with arguments and without argument" {
+   set current_info_period [lindex [S 0 SENTINEL DEBUG] 1]
+   S 0 SENTINEL DEBUG info-period 8888
+   assert { [lindex [S 0 SENTINEL DEBUG] 1] == {8888} }
+   S 0 SENTINEL DEBUG info-period $current_info_period
+   assert { [lindex [S 0 SENTINEL DEBUG] 1] == $current_info_period }
```

Comment:
general comment, we usually prefer to use `assert_equal` since on failure it'll print the mismatching values, and not just an indication of the fact it failed.

Reply:
Address your comments, update to assert_equal to follow the prefer way in test case. Please check it., and there is no documentation to fix, in the json file, the summy is already there

Reply:
by "documentation fix" i meant that this PR doesn't affect redis itself.
it just affects the output of COMMAND DOCS, and the content we'll show in redis.io